outputs/outputs.go

```diff
@@ -23,9 +23,17 @@ var (
 		},
 	}
 
-	// Status describes readings with status outputs.
-	Status = sdk.OutputType{
-		Name: "status",
+	// StatusInt describes readings with status outputs where status is an
+	// integer. Status is a very generic output and avoids a plethora of more
+	// specific output types.
+	StatusInt = sdk.OutputType{
+		Name: "status-int",
+	}
+
+	// StatusSting describes readings with status outputs where status is a
```

Comment:
StatusSting 🐝

Reply:
HA! Good example of what gometalinter used to complain about that golangci-lint does not. Thanks!